packages/smooth_app/lib/pages/onboarding/consent_analytics_page.dart

```diff
@@ -80,58 +93,88 @@ class ConsentAnalytics extends StatelessWidget {
     );
   }
 
-  BottomAppBar _buildBottomAppBar(
-      BuildContext context, AppLocalizations appLocalizations) {
-    return BottomAppBar(
-      child: ButtonBar(
-        alignment: MainAxisAlignment.spaceBetween,
-        mainAxisSize: MainAxisSize.max,
-        children: <Widget>[
-          _buildButton(
-            context,
-            appLocalizations.refuse_button_label,
-            const Icon(
-              Icons.close_rounded,
-            ),
-            false,
+  Widget _buildBottomAppBar(
+    final BuildContext context,
+    final AppLocalizations appLocalizations,
+  ) {
+    final Size screenSize = MediaQuery.of(context).size;
+    // Side padding is 8% of total width.
+    final double sidePadding = screenSize.width * .08;
+    return Column(
+      children: <Widget>[
+        Container(
+          height: SMALL_SPACE,
+          width: screenSize.width,
+          color: LIGHT_GREY_COLOR,
+        ),
+        Container(
+          padding: EdgeInsets.symmetric(
+            vertical: VERY_LARGE_SPACE,
+            horizontal: sidePadding,
           ),
-          _buildButton(
-            context,
-            appLocalizations.authorize_button_label,
-            const Icon(
-              Icons.check_rounded,
-            ),
-            true,
+          width: screenSize.width,
+          color: backgroundColor,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
```

Comment:
Don't know if its exactly the way we want it but what about spaceAround that the content doesn't stick to the sides

Reply:
There are 2 kinds of onboarding pages:
1. pages with scrolling content (e.g. "health"), where we use `Stack` and a `Positioned` bottom bar
2. pages without scrolling content (e.g. "analytics"), where we use a `Column`

For the `Column` cases, the idea is to use `spaceBetween` to glue the bar to the bottom. And let flutter center the rest, heght-wise.